web/src/components/apps/DashboardVersionCard.jsx

```diff
@@ -454,6 +455,12 @@ class DashboardVersionCard extends React.Component {
   }
 
   deployVersion = (version, force = false, continueWithFailedPreflights = false, redeploy = false) => {
+    if (this.props.isHelmManaged) {
```

Comment:
if in helm managed mode then we show the helm managed modal and do nothing else.